src/OpenSpaceToolkit/Physics/Environment/Gravitational/Earth.cpp

```diff
@@ -86,6 +99,18 @@ Vector3d                        Earth::Impl::getFieldValueAt                (
 
     (void) anInstant ; // Temporal invariance
 
+    if (type_ == Earth::Type::Spherical)
```

Comment:
This was done here instead of calling Earth::Spherical because something is partially broken in Earth::Spherical when you construct it from the Earth.cpp file, so its simpler to calculate the Two Body gravity as this.